components/infobox/wikis/apexlegends/infobox_map_custom.lua

```diff
@@ -0,0 +1,94 @@
+---
+-- @Liquipedia
+-- wiki=apexlegends
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Map = require('Module:Infobox/Map')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local String = require('Module:StringUtils')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+
+local CustomMap = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _map
+
+function CustomMap.run(frame)
+	local map = Map(frame)
+	_map = map
+	_args = map.args
+	map.addToLpdb = CustomMap.addToLpdb
+	map.createWidgetInjector = CustomMap.createWidgetInjector
+	return map:createInfobox(frame)
+end
+
+function CustomMap:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{
+		name = 'Game Mode(s)',
+		content = {_args.gamemode}
+	})
+	local spantext = ''
+	if String.isNotEmpty(_args.spanstart) then
+		spantext = spantext .. ' - '
+	end
+	table.insert(widgets, Cell{
+		name = 'Played in ALGS',
+		content = {(_args.spanstart or '<i><b>Not </b></i>') .. spantext .. (_args.spanend or '<i><b>Currently</b></i>')}
+	})
+	if String.isNotEmpty(_args.ring) then
+		local ringTable = mw.html.create('table')
+			:addClass('wikitable wikitable-striped wikitable-bordered')
+			:css('width', '325px')
```

Comment:
```suggestion
			:css('width', '100%')
```
should do what you want and is better for mobile display imo

Reply:
Ah yes, perfect. Meant to ask if there was a better way to do that